src/solvers/dgmulti/flux_differencing.jl

```diff
@@ -35,7 +35,8 @@
 end
 
 @inline function hadamard_sum!(du, A, volume_flux, orientation, u,
-                               equations, sparsity_pattern::AbstractSparseMatrix{Bool})
+                               equations, sparsity_pattern::AbstractSparseMatrix{Bool},
+                               skip_index = nothing)
```

Comment:
```suggestion
                               equations, sparsity_pattern::AbstractSparseMatrix{Bool})
```
`skip_index` seems to be unused

Reply:
It is, but I introduced it to match the function signature for dispatch in https://github.com/trixi-framework/Trixi.jl/blob/a95136704549584b519a6096b3f0bcab1010e348/src/solvers/dgmulti/flux_differencing.jl#L258. 

The type of `sparsity_pattern` toggles dispatch and overrides `skip_index` if `sparsity_pattern != nothing`